web/styles/message_header.css

```diff
@@ -6,13 +6,13 @@
     text-overflow: ellipsis;
     white-space: nowrap;
     font-weight: 600;
-    font-size: calc(15em / 14);
-    line-height: 120%;
     position: relative;
     z-index: 0;
     background-color: var(--color-background);
 
     .message-header-contents {
+        font-size: 1.0714em; /* 15px at 14px em */
```

Comment:
This font-size change probably changes the em-value on the `height` property below.

Reply:
In fact this may be why the before and after shots at e.g., 16px look off to me. This also throw into question whether 14px is the right em value to target for these adjustments, or 16px--similar to the nav bar.

Reply:
Wait is this a font-size change? I was under the impression that `calc(15em / 14)` is functionally exactly the same as `1.0714em`

Reply:
But the height should be based on 15px, right? I believe that's the followup needed from https://github.com/zulip/zulip/pull/32933/files#r1910137015

Reply:
It's functionally the same, yes, but any time `font-size` is set to anything other than `1em`, that changes the em value for other properties in the style block and its children.

Reply:
Right, but since the parent already had this font-size in em then it was still true that this style block and its children had the different em value.

Reply:
I'm talking about the `height: 2em; /* 28px at 14px em */`...perhaps what's needed there is just the 15px calculation, meaning the comment should be `/* 30px at 15px/1em */`

Reply:
Ah yeah, definitely the height issue is messed up, but that happened already in a previous PR and this PR doesn't exacerbate that issue. I see this PR as a code differential with no visible differences, and #33192 as fixing a bug introduced in #32933. So I think it makes sense to continue that discussion there, unless I'm still missing something.